src/webvr-polyfill.js

```diff
@@ -95,6 +95,18 @@ WebVRPolyfill.prototype.enablePolyfill = function() {
 
   // Provide the VRDisplay object.
   window.VRDisplay = VRDisplay;
+
+  // Provide vrEnabled
+  var that = this;
+  Object.defineProperty(navigator, 'vrEnabled', {
+    get: function () {
+      return that.isCardboardCompatible() &&
+        (document.fullscreenEnabled ||
```

Comment:
I'm not sure I'm following you. I haven't bothered to implement anything here for the deprecated API. I could do that if you think it's useful. But from the spec, I don't think `vrEnabled` requires any actual devices to be connected. I think permission on the context is sufficient, which would mean 1) having the required iframe attributes and 2) HTTPS. Full screen permission should be a valid proxy for both of those.